setup.cfg

```diff
@@ -183,6 +183,8 @@ ignore_errors = True
 ignore_errors = True
 [mypy-pytorch_lightning.utilities.cli]
 ignore_errors = False
+[mypy-pytorch_lightning.utilities.device_parser]
```

Comment:
doesn't it make more sense to just list the files here where we want to exclude it rather then excluding everything and then reinclude some?

Reply:
Maybe it's a good question for @Borda?

Reply:
Depends on the number of files to include vs exclude

The current approach is shorter given the current typing state